src/microceph.py

```diff
@@ -154,6 +154,8 @@ def remove_disk_cmd(osd_num: int, force: bool = False) -> None:
     cmd = ["microceph", "disk", "remove", str(osd_num)]
     if force:
         cmd.append("--bypass-safety-checks")
+        # prevent crush scaledown for juju storage removal.
```

Comment:
I'm generally ok with this but I feel for future travelers it would make sense to have the rationale why we don't want to scaledown crush here documented

Reply:
ack, added a multiline note to describe it.